index.html

```diff
@@ -9,8 +9,8 @@
             Home
         </title>
     </head>
-    
-    <body class="home">
+
+    <body class="home background_Image">

```

Comment:
check the class name with the CSS file.

Reply:
> check the class name with the CSS file.

Its correct ✔️  and its working fine too✌🏼